src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,60 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.Objects;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static int MAX_ARRAY_VALUE = 10;
+    private K key;
+    private V value;
+    private K[] keyArray = (K[]) new Object[MAX_ARRAY_VALUE];
```

Comment:
Java має статичну типізацію. У нас не має необхідності вказувати тип змінної в назві тому, що ми і так знаємо що це масив
```suggestion
    private K[] keys = (K[]) new Object[MAX_ARRAY_VALUE];
```

Reply:
Переименую) Думал, так понятнее